pyproject.toml

```diff
@@ -83,7 +83,9 @@ dev = [
 gcp = [
     "cloud-tpu-client",  # for creating and managing TPUs.
     "crcmod",  # for file uploads.
-    "kubernetes",
+    # kubernetes==32.0.0 will cause "kube_config.py:520] Object of type ConfigNode is not JSON serializable" error
+    # when loading kube config, hence failing kubernetes client authentication.
```

Comment:
Is there a github issue to track the problem?

Reply:
I just found this one: https://github.com/kubernetes-client/python/issues/2333

Reply:
I will merge this PR now so unblock users. I can file a separate PR to include a link to this github issue

Reply:
https://github.com/apple/axlearn/pull/947